test-base.cfg

```diff
@@ -208,19 +208,26 @@ commands =
 
 # bin/pycodestyle : pycodestyle validation of the packages source.
 [pycodestyle]
-recipe = zc.recipe.egg
+recipe = zc.recipe.egg:scripts
 eggs = pycodestyle
-args = ['--exclude=tests', '--config=${pycodestyle-cfg:cfgfile}']
```

Comment:
As far as I understood tests should not be excluded for `pycodestyle`, but only for `pydocstyle`.

I considered that an accidental remnant to be sweeped up.